packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -42,19 +39,21 @@ type FormData = {
 
 // eslint-disable-next-line no-empty-pattern
 const ResetPassword = ({
-  serviceName,
+  // TODO: Pull service name from Relier model FXA-6437
```

Comment:
I'm guessing you intend to update this in other reset PW tickets assigned this sprint 👍